src/OpenTelemetry.Exporter.OpenTelemetryProtocol/BaseOtlpExporterOptions.cs

```diff
@@ -0,0 +1,160 @@
+// <copyright file="BaseOtlpExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETFRAMEWORK
+using System.Net.Http;
+#endif
+using System.Reflection;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// Base otlp exporter options.
+    /// </summary>
+    internal abstract class BaseOtlpExporterOptions
```

Comment:
Is it possible to use the existing `OtlpExporterOptions` class as the base rather than introducing a new one?

Reply:
Have not tried that. Any particular reason we would want to use existing one? I kept everything separate so that if we need to add/remove options we could without affecting the existing options class.

Reply:
The individual exporter classes have public constructors that accept an `OtlpExporterOptions`. Given this, do you think using it as a base class might be good? For example, we might be able to avoid the need to add additional constructor overloads accepting `OtlpTraceExporterOptions` and/or `BaseOtlpExporterOptions`.

Reply:
@alanwest - Have opened https://github.com/open-telemetry/opentelemetry-dotnet/pull/4490 that uses `OtlpExporterOptions` as base.